packages/trader/src/sass/app/_common/mobile-dialog.scss

```diff
@@ -8,8 +8,8 @@ $shadow-cover-index: 3;
 $shadow-index: 2;
 $content-index: 1;
 
-/** @define fullscreen-dialog; weak */
-.fullscreen-dialog {
+/** @define mobile-dialog; weak */
```

Comment:
Question: Shouldn't we move this file out to components as well?

Reply:
@reza-fs since we are not supporting mobile right now. i believe this will change later. it can be done when we do mobile.